prime-router/src/testIntegration/kotlin/datatests/mappinginventory/TranslateAndCompareFile.kt

```diff
@@ -0,0 +1,21 @@
+package gov.cdc.prime.router.datatests.mappinginventory
+
+import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.cli.tests.CompareData
+import gov.cdc.prime.router.datatests.TranslationTests
+
+class TranslateAndCompareFile {
```

Comment:
In kotlin, we can just have a stand-alone function in this file, we don't need a class to contain it.